sagemaker-pipeline-multi-model/restate-project.ipynb

```diff
@@ -2,31 +2,31 @@
  "cells": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    endpoint_name = "restate-staging"
</pre><p>I see you removed the timestamp from the endpoint - I think you want to keep that so there can be a unique endpoint name to avoid any naming conflicts?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3500/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Reply:
Updated the notebook per feedback, change in commit https://github.com/aws/amazon-sagemaker-examples/pull/3500/commits/45da7dcd9c094c169d2f8f4d0d11052e96c83000. Thank you!